doc/source/GettingStarted/SettingUpGafferCommand/index.md

```diff
@@ -0,0 +1,142 @@
+# Setting Up the "gaffer" Command #
+
+After you have installed Gaffer, it will remain a collection of files and directories on your file system. Because it is not yet configured as a command, you must navigate to its directory every time. This could become very tedious, so we recommend that you modify your `PATH` environment variable to allow access to the `gaffer` command from anywhere in the terminal.
+
+> Note :
+> For these instructions, we will assume you have Gaffer installed to the `/opt/` directory. If you have installed it elsewhere, replace `/opt/` with the directory you installed it to.
+
+
+### Environment Variables ###
```

Comment:
Here's what I said to John about this:

> In my experience (interacting with users at my old jobs, as well as talking to artists here) there are very, very few day-to-day (a.k.a. "rote") users who would immediately react to the words "environment variable" by thinking "I know what those are and what is involved in setting them up." I asked IE's LookDev lead what he thought of configuring environment variables, and he admitted it was foreign territory for him. That, combined with finding a pretty abysmal lack of coverage by the Linux community on how to _permanently_ set up an env var that didn't devolve into navel-gazing debates on which shell configuration file to use, motivated my fleshing out that article.

So, yeah, not super revelatory to other studio techs, but probably helpful to people trying it out on their home machine. This will become a moot point once we have an interactive installer, but I don't think it's actively harmful to spent two paragraphs establishing what we're talking about before prescribing to the public to mess with their env vars.